python/ray/autoscaler/updater.py

```diff
@@ -226,15 +226,13 @@ def do_update(self):
 
         m = "{}: Initialization commands completed".format(self.node_id)
         with LogTimer("NodeUpdater: {}".format(m)):
-            with open("/dev/null", "w") as redirect:
-                for cmd in self.initialization_commands:
-                    self.ssh_cmd(cmd, redirect=redirect)
+            for cmd in self.initialization_commands:
+                self.ssh_cmd(cmd)
```

Comment:
No configurable?

Reply:
No. If you want to disable logging, just add >/dev/null to your setup command.

No need to leave foot-guns for the user.